portfolio/src/main/webapp/index.html

```diff
@@ -31,13 +31,25 @@ <h1>Lily Zhou</h1>
         <button class="button"><a href="images.html">Food Journal</a></button>
         <button class="button"><a href="projects.html">Projects</a></button>
       </div>
-      <form action="/data" method="POST">
-        <textarea name="name-input">Name</textarea>
-        <textarea name="comment-input">Your comment here.</textarea>
-        <br/><br/>
-        <input type="submit" />
-      </form>
-      <div id="comments-container"></div>
+      <div class="comments-box">
+        <form action="/data" method="POST">
+          <label for="name-input" class="comment-label">Name: </label><br>
+          <input type="text" name="name-input"><br><br>
+          <label for="comment-input" class="comment-label" rows="4" cols="100">Comment: </label><br>
+          <textarea name="comment-input" class="text-input"></textarea><br><br>
+          <input type="submit" />
+        </form>
+        <form action="/data">
+          <br><br>
+          <label for="comment-input" class="comment-label">Max Comments Displayed: </label>
+          <input type="number" id="max-input" name="max-input" min="0" max="100" value="5"><br><br>
```

Comment:
I think you probably want to move this out of the form, since it's not part of the leaving-a-comment section, but part of the displayed comments section.